.travis.yml

```diff
@@ -31,6 +31,11 @@ language: python
 python:
   - 2.7
 
+env:
+  - PYTEST="tests/test_views.py"
+  - PYTEST="tests/test_workflows.py"
+  - PYTEST="--ignore tests/test_views.py --ignore tests/test_workflows.py"
```

Comment:
I don't like the harcoded test filenames. A better solution (IMO) is to separate the tests in `integration` and `unit` folders, and run those separately.